mautic/mautic
# Bug fix for published labels

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [X]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

This PR fixes the labels statuses. Opted to remove the text label and use only colors since now we have active/available statuses and Mautic code not always differentiate them

![image](https://github.com/mautic/mautic/assets/116097999/75416411-1a54-43cb-8e0b-e1a630add538)

![image](https://github.com/mautic/mautic/assets/116097999/dfae82b1-202c-4fbf-85a6-fa04f30028a8)





#### Steps to test this PR:
1. Assets > Create new and see 
2. Save & Close and see the other

![image](https://github.com/mautic/mautic/assets/116097999/3dd65be3-fa10-4cff-9ece-8d91403a8ad0)


![image](https://github.com/mautic/mautic/assets/116097999/3bd494a4-32f7-4cc3-b184-98254a54a09d)
